tests/schema/models_postgres_index.py

```diff
@@ -19,6 +19,7 @@ class Index(Model):
     sp_gist = fields.CharField(max_length=200)
     hash = fields.CharField(max_length=200)
     partial = fields.CharField(max_length=200)
+    array = ArrayField(element_type="text", default=["a", "b", "c"])
```

Comment:
The `Index` model is used for testing indexes specifically and there is no index on array in this case. It would be better to use another model for testing arrays.

It might make sense to add a new test or find a more suitable one.

Reply:
Hi @henadzit, sorry for the late fix, I've move those to new tests and make some more changes